netgames/CMakeLists.txt

```diff
@@ -1,11 +1,13 @@
 include_directories("includes")
 add_subdirectory(dmfc)
-add_subdirectory(anarchy)
-add_subdirectory(coop)
-add_subdirectory(ctf)
-add_subdirectory(entropy)
-add_subdirectory(hoard)
-add_subdirectory(hyperanarchy)
-add_subdirectory(monsterball)
-add_subdirectory(roboanarchy)
-add_subdirectory(tanarchy)
+
+# Don't build these yet
+#add_subdirectory(anarchy)
+#add_subdirectory(coop)
+#add_subdirectory(ctf)
+#add_subdirectory(entropy)
+#add_subdirectory(hoard)
+#add_subdirectory(hyperanarchy)
+#add_subdirectory(monsterball)
+#add_subdirectory(roboanarchy)
+#add_subdirectory(tanarchy)
```

Comment:
Why this code commented out?

Reply:
Because we are not building netgames yet. If we want to build them alongside dmfc, we need to fix them because when I last tried building them, they were broken (like d3c was).

Reply:
Well, I think, commenting them out is not related to intended partially reverting dmfc changes. Let's keep them building and resolve issues as they appears.

Reply:
Sure, and if I did the opposite someone else would complain that my PR ships broken code. This makes dmfc work with existing game assets. I advise a separate PR for fixing d3ms and subsequently turning them back on.